core-services/prow/02_config/openshift/cluster-logging-operator/_prowconfig.yaml

```diff
@@ -34,78 +22,14 @@ tide:
     - needs-rebase
     repos:
     - openshift/cluster-logging-operator
-  - includedBranches:
-    - openshift-4.9
-    - release-4.9
-    labels:
-    - approved
-    - bugzilla/valid-bug
-    - lgtm
-    - staff-eng-approved
-    missingLabels:
-    - backports/unvalidated-commits
-    - bugzilla/invalid-bug
-    - do-not-merge/hold
-    - do-not-merge/invalid-owners-file
-    - do-not-merge/work-in-progress
-    - needs-rebase
-    repos:
-    - openshift/cluster-logging-operator
-  - includedBranches:
-    - main
```

Comment:
> you rely on the "everything else" query (the one with excludedBranches ) for these

This removes all config for other branches but based on the slack comment I'm not sure I understand how "everything else" is gated